js/adapt-contrib-instructionError.js

```diff
@@ -29,6 +40,28 @@ class InstructionError extends Backbone.Controller {
     notify.popup(notifyObject);
   }
 
+  showInlineError(model) {
+    const data = model.toJSON();
+    model.toggleClass('has-error', true);
+
+    // Update instruction text
+    const dataWithInitial = Object.assign(data, { instruction: model.get('instructionInitial') });
+    const instruction = Handlebars.compile(this.config.body)(dataWithInitial);
+    model.set('instruction', instruction);
+
+    // Focus on instruction text element
+    const $instruction = $(`.${data._id}`).find('.component__instruction').first();
```

Comment:
```suggestion
    const $instruction = $(`[data-adapt-id=${data._id}]`).find('.component__instruction').first();
```

Use data-adapt-id instead of class. Some of the ATT ids are invalid classes.

![image](https://github.com/user-attachments/assets/6effbd84-ddab-4715-84c5-a48795719de0)